package.json

```diff
@@ -180,6 +181,10 @@
     "expo-random": "*",
     "history": "^5.3.0",
     "lodash": "^4.17.21",
-    "react-native-keyboard-aware-scroll-view": "^0.9.5"
+    "moment": "^2.29.4",
```

Comment:
Moment have vulnerabilities is all versions except the latest. Since we are only using very little of it we could avoid it by using something like this instead

`var options = { year: 'numeric', month: 'short', day: 'numeric' };
var today  = new Date();

console.log(today.toLocaleDateString("en-US", options));`

Reply:
We can create a Date component in MEC. It is needed and this is a good time to implement it. Setting this issue to blocked until it is resolved